tests/testthat/helper.R

```diff
@@ -0,0 +1,6 @@
+library(mlr3)
+library(mlr3cluster)
+library(checkmate)
+library(testthat)
+
+lapply(list.files(system.file("testthat", package = "mlr3cluster"), pattern = "^helper.*\\.[rR]", full.names = TRUE), source)
```

Comment:
Note that this sources files from "[mlr3cluster]/inst/testthat". You can remove this line.

Reply:
Ok, that makes sense.